tests/syft/api/load_lib_test.py

```diff
@@ -0,0 +1,9 @@
+# syft absolute
+import syft as sy
+
+
```

Comment:
Could we get another test that checks for different errors when an invalid string or an invalid object is passed?

Reply:
Added error testing.